ckanext/canada/tables/aistrategy.yaml

```diff
@@ -0,0 +1,818 @@
+dataset_type: aistrategy
+
+title: AI Strategy Implementation Tracker
+notes: Access, upload and modify the AI Strategy Implementation Tracker for your organization
+
+template_version: 3
+template_updated: 2025-08-04
+
+portal_type: info
+collection: pd
+
+resources:
+- title: AI Strategy Implementation Tracker
+  resource_name: aistrategy
+
+  published_resource_id: # TBD
+
+  create_form: true
+  edit_form: true
+
+  fields:
+
+  # DEP 1-1
+  - datastore_id: ref_number
+    label:
+      en: Reference Number
+      fr: Numéro de référence
+    description:
+      en: The unique number assigned to a service using AI to make it easier to refer to specific services.
+      fr: Le numéro unique attribué à un service à l'aide de l'IA pour faciliter le référencement à des services précis.
+    obligation: Mandatory
+    excel_required: true
+    form_required: true
+    format_type: Free text
+    visible_to_public: Yes
+    occurrence: Single
+    datastore_type: text
+    excel_error_formula: 'OR({default_formula},LEN({cell})-LEN(SUBSTITUTE({cell},",",""))>0)'
+    validation:
+      en: |
+        This field must not be empty.
+        This field cannot contain commas.
+      fr: |
+        Ce champ ne doit pas être vide.
+        Ce champ ne peut pas contenir de virgules.
+
+  # DEP 1-2
+  - datastore_id: reporting_period
+    label:
+      en: Reporting Period
+      fr: Période de déclaration
+    description:
+      en: Year and month of the update
+      fr: Année et mois de la mise à jour
+    obligation: Mandatory
+    excel_required: true
+    form_required: true
+    format_type: Controlled List
+    datastore_type: text
+    validation:
+      en: Must match expected completion options in the controlled list.
+      fr: Doit correspondre aux options d'achèvement prévues de la liste contrôlée.
+    occurrence: Single
+    excel_full_text_choices: true
```

Comment:
I suggest setting this to false here for easier data entry

Reply:
this is a style thing though, see what you prefer in the generated excel template